src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+                <code>grid-column-start, grid-column-end, grid-row-start, grid-row-end</code>
+                <br>
+                <p><b>Values</b></p>
+                <ul>
+                    <li>line: number or name of grid line</li>
+                    <li>span &lt;number&gt;: item spans across given number of tracks</li>
+                    <li>span &lt;name&gt;: item spans until given track</li>
+                    <li>auto: auto-placement</li>
+                </ul>
+                <p class="fragment">Item will span 1, if no grid-column-end/grid-row-end is declared. Items can overlap each other. Use z-index to control stacking order.</p>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+              <code class="formattedCode">
+                # CSS
+                  .grid-container {
+                    grid-template-columns: [l1] 140px [l2] 150px [l3] auto [l4] 150px [l5] 140px [end];
+                    grid-template-rows: [r1-start] 25% [r1-end r2-start] 100px [r2-end r3-start] auto [r3-end];
+                  }
+
+                  .grid-item-1 {
+                    grid-column-start: 1;
+                    grid-column-end: span l4;
+                    grid-row-start: 2;
+                    grid-row-end: span 1;
+                  }
+      
+                # HTML
+                <div class="grid-container">
+                  <div class="grid-item-1">Item 1</div>
+                  <div class="grid-item-2">Item 2</div>
+                  <div class="grid-item-3">Item 3</div>
+                  <div class="grid-item-4">Item 4</div>
+                  <div class="grid-item-5">Item 5</div>
+                </div>
+        </code><br><div class="fragment grid-container grid-15">
+          <div class="grid-item-1">Item 1</div>
+          <div class="grid-item-2">Item 2</div>
+          <div class="grid-item-3">Item 3</div>
+          <div class="grid-item-4">Item 4</div>
+          <div class="grid-item-5">Item 5</div>
+      </div></pre>
+            </section>
+
+            <section>
+                <h3>Item Location - Shorthand</h3>
+                <code>grid-column, grid-row</code>
+                <br>
+                <pre><code class="formattedCode css">
+                .item {
+                  grid-column: &lt;start-line&gt; / &lt;end-line&gt; | &lt;start-line&gt; / span &lt;value&gt;;
+                  grid-row: &lt;start-line&gt; / &lt;end-line&gt; | &lt;start-line&gt; / span &lt;value&gt;;
+                }
+              </code></pre>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+            <code class="formattedCode css">
+                .grid-item-1 {
+                  <!-- grid-column-start: 1; -->
+                  <!-- grid-column-end: span l4; -->
+                  <!-- grid-row-start: 2; -->
+                  <!-- grid-row-end: span 1; -->
```

Comment:
These comments should be with #

Reply:
Yes, that happed by VS code making that automatically... I will change it :)

Reply:
[Fixed](https://github.com/ReDI-Munich-Frontend/fed-munich-2021-spring/commit/1c4091a9316975c61acb733fbfdb941f98d3266a)